client/settings/settings-tab-panel/index.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import { TabPanel } from '@wordpress/components';
+import { getQuery } from '@woocommerce/navigation';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import PaymentMethodsPanel from '../payment-methods';
+import PaymentSettingsPanel from '../payment-settings';
+
+// This grabs the "panel" URL query string value to allow for opening a specific tab.
+const { panel } = getQuery() || '';
```

Comment:
`getQuery` returns an object - this is attempting to restructure a string - the equivalent of `''.panel`

```suggestion
const { panel } = getQuery() || {};
```

In which case would `getQuery()` return a falsy value? it looks like it always returns a consistent type: https://github.com/woocommerce/woocommerce-admin/blob/main/packages/navigation/src/index.js#L165-L171

```suggestion
const { panel } = getQuery();
```